podcasts/ManageDownloadsBannerView.swift

```diff
@@ -0,0 +1,91 @@
+import SwiftUI
+import PocketCastsUtils
+import Combine
+
+class ManageDownloadsBannerModel: ObservableObject {
+
+    @Published var sizeOccupied: String = ""
+
+    let onManageTap: (() -> ())?
+
+    init(initialSize: String, onManageTap: (() -> ())? = nil) {
+        _sizeOccupied = .init(initialValue: initialSize)
+        self.onManageTap = onManageTap
+        loadData()
+    }
+
+    func loadData() {
+        Task { [weak self] in
+            var totalSize = UInt64(0)
+            totalSize += EpisodeManager.downloadSizeOfUnplayedEpisodes(includeStarred: true)
+            totalSize += EpisodeManager.downloadSizeOfInProgressEpisodes(includeStarred: true)
+            totalSize += EpisodeManager.downloadSizeOfPlayedEpisodes(includeStarred: true)
+            let sizeAsStr = SizeFormatter.shared.noDecimalFormat(bytes: Int64(totalSize))
+            Task { @MainActor in
```

Comment:
If you need to run it on the main actor, what about `await MainActor.run {` instead?

Reply:
TIL! I feel there is too much options in the new async features in Swift!